src/cli/commands/version.js

```diff
@@ -124,6 +124,8 @@ export async function setVersion(
       // add manifest
       await spawn('git', ['add', pkgLoc]);
 
+      await config.executeLifecycleScript('version');
```

Comment:
This will only execute if the package is in a Git repo, as it's inside the `if (isGit) {` condition. You probably want to execute it when outside of a Git repo too, right?